<div class="entry-content">
 <div class="wp_orcid_post">
  <a href="http://orcid.org/0000-0002-2747-368X" rel="author" target="_blank">
   0000-0002-2747-368X
  </a>
 </div>
 <p>
  Sequencing results from the samples that were
  <a href="http://onsnetwork.org/sjwfriedmanlab/2015/10/23/sanger-sequencing-submission-pcr2-1rlov-clones/" target="_blank" title="Sanger Sequencing Submission – pCR2.1/RLOv Clones">
   submitted to Genewiz on Friday
  </a>
  have come back:
 </p>
 <ul>
  <li>
   SW01    RLOv_DNA_Helicase-M13F_-21_
  </li>
  <li>
   SW02    RLOv_head_to_tail-M13F_-21_
  </li>
  <li>
   SW03    RLOv_membrane_gene_1-M13F_-21_
  </li>
  <li>
   SW04    RLOv_membrane_gene_2-M13F_-21_
  </li>
  <li>
   SW05    RLOv_tail_fiber-M13F_-21_
  </li>
  <li>
   SW06    RLOv_DNA_Helicase-M13R
  </li>
  <li>
   SW07    RLOv_head_to_tail-M13R
  </li>
  <li>
   SW08    RLOv_membrane_gene_1-M13R
  </li>
  <li>
   SW09    RLOv_membrane_gene_2-M13R
  </li>
  <li>
   SW10    RLOv_tail_fiber-M13R
  </li>
 </ul>
 <p>
  The data (10-313205054_ab1.zip) has been stored in the following location: backupordie/sequencing_data/Sanger.
 </p>
 <p>
  Sequences were loaded into Geneious (v.9.0.2). Vector sequences were trimmed/annotated using the Trim Ends with UniVec feature in Geneious.
 </p>
 <p>
  Each clone was sequenced once from each direction, so the two sequences generated from each clone were mapped to the original sequence from which the primers were designed using Geneious Mapper.
 </p>
 <p>
  The Geneious analysis was exported and saved in the following location:
 </p>
 <p>
  backupordie/Sam/Sequencing_Analysis/Sanger/20151026_RLOv_clones_Sanger_analysis.geneious
 </p>
 <p>
  <span style="text-decoration: underline;">
   Results
  </span>
  :
 </p>
 <p>
  Each clone’s sequence matches that of the source sequence, so we’re good to go!
 </p>
 <p>
  Will proceed with dye-based quantification of each plasmid. Will then proceed with developing ISH probes (membrane genes 1 &amp; 2, tail fiber gene) or qPCR standard curves (DNA helicase, head-to-tail).
 </p>
 <p>
  In the alignments below, the reference sequence is highlighted in light yellow. The two electropherograms are align below the reference. The grey line in the consensus sequence indicates any sequence disagreements by placement of a black mark at the position. However, the sequences all match, so there are no black marks in the regions between the identified vector sequences (red annotations below each electropherogram).
 </p>
 <p>
  <a data-slb-active="1" data-slb-group="830" href="https://dl.dropbox.com/s/4ffdxo2tl45kayz/20151026_RLOv_DNA_helicase_sanger.png?dl=0">
   <img alt="" class="alignnone" height="182" src="images/20151026_RLOv_DNA_helicase_sanger.png" width="1044"/>
  </a>
 </p>
 <p>
  <a data-slb-active="1" data-slb-group="830" href="https://dl.dropbox.com/s/x0qpbvwyn7dgwan/20151026_RLOv_head_to_tail_sanger.png?dl=0">
   <img alt="" class="alignnone" height="223" src="images/20151026_RLOv_head_to_tail_sanger.png" width="1050"/>
  </a>
 </p>
 <p>
  <a data-slb-active="1" data-slb-group="830" href="https://dl.dropbox.com/s/pgw1tydf1rj9bi9/20151026_RLOv_membrane_gene_1.png?dl=0">
   <img alt="" class="alignnone" height="214" src="images/20151026_RLOv_membrane_gene_1.png" width="1054"/>
  </a>
 </p>
 <p>
  <a data-slb-active="1" data-slb-group="830" href="https://dl.dropbox.com/s/tub410zqteyygca/20151026_RLOv_membrane_gene_2.png?dl=0">
   <img alt="" class="alignnone" height="220" src="images/20151026_RLOv_membrane_gene_2.png" width="1059"/>
  </a>
 </p>
 <p>
  <a data-slb-active="1" data-slb-group="830" href="https://dl.dropbox.com/s/ucl3sw9j6u7kjca/20151026_RLOv_tail_fiber.png?dl=0">
   <img alt="" class="alignnone" height="224" src="images/20151026_RLOv_tail_fiber.png" width="1063"/>
  </a>
 </p>
 <div class="addtoany_share_save_container addtoany_content_bottom">
  <div class="a2a_kit a2a_kit_size_32 addtoany_list a2a_target" id="wpa2a_3">
   <a class="a2a_button_facebook" href="http://www.addtoany.com/add_to/facebook?linkurl=http%3A%2F%2Fonsnetwork.org%2Fsjwfriedmanlab%2F2015%2F10%2F26%2Fsanger-sequencing-analysis-pcr2-1rlov-clones%2F&amp;linkname=Sanger%20Sequencing%20Analysis%20%E2%80%93%20pCR2.1%2FRLOv%20Clones" rel="nofollow" target="_blank" title="Facebook">
   </a>
   <a class="a2a_button_twitter" href="http://www.addtoany.com/add_to/twitter?linkurl=http%3A%2F%2Fonsnetwork.org%2Fsjwfriedmanlab%2F2015%2F10%2F26%2Fsanger-sequencing-analysis-pcr2-1rlov-clones%2F&amp;linkname=Sanger%20Sequencing%20Analysis%20%E2%80%93%20pCR2.1%2FRLOv%20Clones" rel="nofollow" target="_blank" title="Twitter">
   </a>
   <a class="a2a_button_google_plus" href="http://www.addtoany.com/add_to/google_plus?linkurl=http%3A%2F%2Fonsnetwork.org%2Fsjwfriedmanlab%2F2015%2F10%2F26%2Fsanger-sequencing-analysis-pcr2-1rlov-clones%2F&amp;linkname=Sanger%20Sequencing%20Analysis%20%E2%80%93%20pCR2.1%2FRLOv%20Clones" rel="nofollow" target="_blank" title="Google+">
   </a>
   <a class="a2a_dd addtoany_share_save" href="https://www.addtoany.com/share_save">
   </a>
  </div>
 </div>
</div>
